.github/workflows/ci.yml

```diff
@@ -216,7 +216,7 @@ jobs:
 
       - name: Run Playwright tests
         if: ${{ success() || failure() }}
-        run: yarn workspace @blockprotocol/site playwright test --project=integration
+        run: yarn workspace @blockprotocol/site playwright test --project=integration-chrome --project=integration-firefox --project=integration-safari --project=integration-iphone --project=integration-pixel
```

Comment:
It'll be nice to use wildcard project name once this is supported upstream: https://github.com/microsoft/playwright/issues/15128

Reply:
Nice!